web/templates/settings/admin_auth_methods_list.hbs

```diff
@@ -5,6 +5,6 @@
       is_checked=enabled
       label=method
       is_disabled=disable_configure_auth_method
-      tooltip_text=unavailable_reason
+      help_icon_tooltip_text=unavailable_reason
```

Comment:
`web/templates/settings/settings_numeric_input.hbs` might want a similar rename for consistency.

Reply:
Added a commit for it. `settings_numeric_input` does not use that field as we do not show tooltips for those inputs as of now.